packages/react/src/components/UIShell/SideNav.js

```diff
@@ -91,14 +92,16 @@ const SideNav = React.forwardRef(function SideNav(props, ref) {
     });
   }
 
-  const eventHanders = {
-    onFocus: event => handleToggle(event, true),
-    onBlur: event => handleToggle(event, false),
-  };
+  let eventHandlers = {};
```

Comment:
just wondering, any reason for the change from `const` to `let` declaration? it doesn't look like the variable is reassigned a different value

Reply:
Hi @emyarod ! And this is true. I changed it as before it would always have at least some content (ie the onFocus/onBlur), and now that was not always the case. I felt this was a (subtle) nod to the fact this objects content could change. I am happy to change it back to a `const` if you prefer.